packages/plugins/apollo-usage-report/src/stats.ts

```diff
@@ -0,0 +1,668 @@
+// Copied from https://github.com/apollographql/apollo-server/blob/8c6579e5b61276b62dc7e30e6fac9a4242e24daa/packages/server/src/plugin/usageReporting/stats.ts
+/* eslint-disable */
+
+import type { NonFtv1ErrorPath } from '@apollo/server-gateway-interface';
```

Comment:
We can avoid this package entirely by copying this too, no?

Reply:
Yes, we can. We discussed it on slack, and we decided that if the Licence allows it, and the dependency is small, it is ok to have it as a dependence.

That's why I've made a copy of the relevant part of the apollo-server code. I don't think it is worth to embed the whole `apollo-server` deps just for a class ^^'